SimPEG/InvProblem.py

```diff
@@ -91,28 +94,28 @@ def startup(self, m0):
 
         self.model = m0
 
-        if isinstance(self.dmisfit, DataMisfit.BaseDataMisfit):
-            print("""
-    SimPEG.InvProblem is setting bfgsH0 to the inverse of the eval2Deriv.
-    ***Done using same Solver and solverOpts as the problem***"""
-            )
-            self.opt.bfgsH0 = self.dmisfit.prob.Solver(
-                self.reg.deriv2(self.model), **self.dmisfit.prob.solverOpts
-            )
-        elif isinstance(self.dmisfit, ObjectiveFunction.BaseObjectiveFunction):
-            for objfct in self.dmisfit.objfcts:
-                if isinstance(objfct, DataMisfit.BaseDataMisfit):
-                    print("""
-    SimPEG.InvProblem is setting bfgsH0 to the inverse of the eval2Deriv.
-    ***Done using same Solver and solverOpts as the {} problem***""".format(
-                            objfct.prob.__class__.__name__
+        if self.bfgs:
+            if isinstance(self.dmisfit, DataMisfit.BaseDataMisfit):
+                print("""SimPEG.InvProblem is setting bfgsH0 to the inverse of the eval2Deriv.
+                      ***Done using same Solver and solverOpts as the problem***"""
+                      )
```

Comment:
sorry - being picky. Would you mind updating bracketing levels here and in the following few lines? So 
```
print("""SimPEG.InvProblem is setting bfgsH0 to the inverse of the eval2Deriv.
          ***Done using same Solver and solverOpts as the problem***"""
)
```